docker-compose.yml

```diff
@@ -32,6 +32,7 @@ services:
         - ./volumes/ssl.crt:/usr/local/tomcat/conf/ssl.crt
         - ./volumes/ssl.key:/usr/local/tomcat/conf/ssl.key
         - ./docker/oscar/index.jsp:/usr/local/tomcat/webapps/ROOT/index.jsp
+        - ./docker/oscar/drugref2:/usr/local/tomcat/bin/drugref
```

Comment:
What's the purpose of all these files?

Reply:
They are scripts to call for drugref. So in #72 , we want to call update to have drugref data.

Reply:
Those could be added in the Dockerfile then. Where possible that's preferred.

Reply:
@ferdinandpolpol , ping